ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -392,7 +392,11 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
                                                                               text: text, labelAccessoryViewMarginLeading: labelAccessoryViewMarginLeading)
 
         let availableWidth = textAreaWidth - (leadingAccessoryAreaWidth + trailingAccessoryAreaWidth)
-        return text.preferredSize(for: font, width: availableWidth, numberOfLines: numberOfLines)
+
+        // The width from preferredSize() will hug the label, which results in cut-off text if the label has additional attributes.
+        // We want to have the available width to prevent any cut-off.
+        return CGSize(width: text.isEmpty ? 0 : availableWidth,
```

Comment:
why is it zero when the text is empty? this seems like a behavioral diffrence?

Reply:
the label shouldn't have any width so that the other views can be placed correctly (this is what the preferredSize() method does, but with fonts. If the font is nil then the width is also 0)